mungegithub/submit-queue/deployment/kubernetes/configmap.yaml

```diff
@@ -24,8 +24,7 @@ data:
     \"pull-kubernetes-kubemark-e2e-gce\",\
     \"pull-kubernetes-e2e-gce-etcd3\",\
     \"pull-kubernetes-bazel\",\
-    \"pull-kubernetes-e2e-kops-aws\",\
-    \"pull-kubernetes-federation-e2e-gce\""
```

Comment:
That's a fair question -- based on previous changes to enable/disable the tests, it appears that the in order to test but not block it has to be done this way: Leave kubernetes-pull-build-test-federation-e2e-gce in submit-queue.presubmit-jobs but remove pull-kubernetes-federation-e2e-gce from test-options.required-retest-contexts.

Perhaps test-infra or @madhusudancs can confirm?